mender-convert

```diff
@@ -14,6 +14,21 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+source modules/bootstrap.sh
+source modules/cliparser.sh
+source modules/decompressinput.sh
+
+###############################################################################
+#                Declaration of important variables for this file             #
+###############################################################################
+declare override_extraargs="" # Override arguments passed to the sub-scripts
+declare compression_type=""   # Detected input file compression, also applied to the output
+declare tfile=""              # Override configuration file options
+declare disk_image=""         # Needed in parse_cli_options, and is passed to decompress_image()
+declare -a overlays=()        # [Dummy] Needed in parse_cli_options, not here
+declare -a overlays=()        # [Dummy] Needed in parse_cli_options, not here
```

Comment:
Duplicate, should the second be `configs`?

Reply:
Yes, sry